### PR TITLE
[doc][yba] Update node OS support

### PR DESCRIPTION
**Pull request by ddhodge**
Update node OS support
DOC-199

@netlify /stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes/

### CODE REVIEW

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md [hunk 1]
```diff
@@ -13,6 +13,34 @@ type: docs
...
+
+Note that support for ARM architectures is unavailable for airgapped setups, because YBA ARM support for [AWS Graviton](https://aws.amazon.com/ec2/graviton/) requires Internet connectivity.
+
+### Supported operating systems
+
+YBA supports deploying YugabyteDB on a variety of [operating systems](../../../reference/configuration/operating-systems/#yugabytedb-anywhere). YBA additionally supports deploying YugabyteDB on Red Hat Enterprise Linux 8. AlmaLinux OS 9 is used by default.
+
+#### Requirements for all OSes
+
+Python 3 is required. If you're using YugabyteDB Anywhere to provision nodes in public clouds, be sure the custom AMI you plan to use has Python 3 installed.
```
Comment (chidmuthu) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md:32]: Can we also put something about how if SELinux is enabled we will also need the corresponding selinux, policycoreutils-python, and python3-libselinux packages. Also their python3 should be accessible by invoking just `python3` (so should be in their path ig, not sure the right wording for that).

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md [hunk 2]
```diff
@@ -13,6 +13,34 @@ type: docs
...
 
+## Prerequisites
+
+Using YugabyteDB Anywhere (YBA), you can deploy YugabyteDB universes on nodes with the following architectures and operating systems.
+
+### Supported CPU architectures
+
+YBA supports deploying YugabyteDB on both x86 and ARM (aarch64) architecture-based hardware.
+
+Note that support for ARM architectures is unavailable for airgapped setups, because YBA ARM support for [AWS Graviton](https://aws.amazon.com/ec2/graviton/) requires Internet connectivity.
```
Comment (chidmuthu) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md:24]: Where are we getting this from? There is no technical limitation to deploying on ARM in airgap, we just require some different packages. Along with rsync/openssl (always required) they will need
libatomic and glibc-langpack-en (for Fedora based operating systems)
libatomic1 and libncurses6 (for Debian based operating systems)

Comment (chidmuthu) [reply docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md:24]: The only ARM limitation is azcopy does not work so you can't backup to Azure Cloud Storage bucket

File: docs/content/stable/reference/configuration/operating-systems.md [hunk 3]
```diff
@@ -39,15 +39,22 @@ Unless otherwise noted, x86 operating systems are supported by all supported ver
 
 | Operating system | x86            | ARM            | Notes |
 | :--------------- | :------------- | :------------- | :--- |
+| AlmaLinux 8      | {{<icon/yes>}} | {{<icon/yes>}} |  |
 | AlmaLinux 9      | {{<icon/yes>}} | {{<icon/yes>}} | Default for YBA-deployed nodes |
```
Comment (chidmuthu) [inline docs/content/stable/reference/configuration/operating-systems.md:43]: AlmaLinux 8 is the default for YBA deployed nodes...

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md (hunk 2 above)
Comment (ddhodge) [reply docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md:24]: https://github.com/yugabyte/yugabyte-db/pull/16951

Comment (ddhodge) [reply docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md:24]: @chidmuthu should we delete this then

Comment (chidmuthu) [reply docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md:24]: Yes, we can remove that part. There is a way for airgapped customers they just will have to manually copy over the DB package.

**Review by chidmuthu — APPROVED** [review]

File: docs/content/preview/reference/configuration/operating-systems.md [hunk 4]
```diff
@@ -35,15 +35,16 @@ Unless otherwise noted, operating systems are supported on all supported version
...
+| AlmaLinux 9      | {{<icon/yes>}} | {{<icon/yes>}} |  |
 | Amazon Linux 2   | {{<icon/yes>}} | {{<icon/yes>}} | v2.18.0 and later |
 | CentOS7          | {{<icon/yes>}} |                | |
-| Oracle 7         | {{<icon/yes>}} |   |
-| Oracle 8         | {{<icon/yes>}} |   |
+| Oracle 7         | {{<icon/yes>}} |                | |
+| Oracle 8         | {{<icon/yes>}} |                | |
 | SUSE Linux Enterprise Server 15 SP4 | {{<icon/yes>}} |   | {{<badge/tp>}} |
 | SUSE Linux Enterprise Server 15 SP5 | {{<icon/yes>}} |   | {{<badge/tp>}} |
 | Ubuntu 18        | {{<icon/yes>}} |                | |
```
Comment (chidmuthu) [inline docs/content/preview/reference/configuration/operating-systems.md:50]: ubuntu also you can use ARM

**Review by chidmuthu — APPROVED** [review]

**Pull request merged**